server/core/store.go

```diff
@@ -319,7 +320,7 @@ func (s *StoreInfo) regionScoreV2(delta int64, deviation int) float64 {
 		score = (K + M*(math.Log(C)-math.Log(A-F+1))/(C-A+F-1)) * R
 	} else {
 		// When remaining space is less then F, the score is mainly determined by available space.
-		score = (K+M*math.Log(C)/C)*R + (F-A)*(K+M*math.Log(F)/F)
+		score = (K+M*math.Log(C)/C)*R + B*(F-A)*(K+M*math.Log(F)/F)
```

Comment:
can we make F configuration and set to a more suitable default value?

Reply:
What is the relationship between `F` and `low-space-ratio`? BTW, I think it is a little bit confusing and misleading.

Reply:
low-space-ratio: avoid other region move in 
F:  it will increase score after it's free disk less than F, so it will move out region to other 
I agree the two args misleading

Reply:
I use max(20,capacity*(1-lowSpaceRatio)) to set F, so it will eliminate F and  low-space-ratio meaning